build_common.sh

```diff
@@ -248,6 +248,9 @@ print_so_deps() {
     Darwin-*)
         run_after_echo otool -L "${1:?}"
         ;;
+    Haiku*)
```

Comment:
`Haiku-*` is supposed to work, does it?

Reply:
You're right we can use `Haiku-*` - I'll update.